techniques/fileDistribution/copyGitFile/1.3/copyFileFromSharedFolder.st

```diff
@@ -110,13 +110,13 @@ bundle agent download_from_shared_folder
         ifvarclass => "!execute_command_${index}";
 
       "@@copyFile@@result_success@@${copyfile[${index}][uuid]}@@Post-modification hook@@${copyfile[${index}][name]}@@${g.execRun}##${g.uuid}@#${copyfile[${index}][destination]} was already in the desired state, so no command was executed"
-        ifvarclass => "execute_command_${index}.copy_file_${index}_kept.!copy_file_${index}_modified";
+        ifvarclass => "execute_command_${index}.!copy_file_${index}_failed.!copy_file_${index}_modified.copy_file_${index}_kept";
```

Comment:
Do I understand correctly that you can have kept + failed at the same time ?
(reading code)
Oh, ok, permissions kept, copy failed - make sense